distribution/docker/src/docker/dockerfiles/default/Dockerfile

```diff
@@ -92,14 +92,15 @@ package_manager,
   "      ${package_manager} update && \n" +
   "      ${package_manager} upgrade -y && \n" +
   "      ${package_manager} install -y --no-install-recommends \n" +
-  "        ca-certificates curl netcat p11-kit unzip zip ${docker_base == 'cloud' ? 'wget' : '' } && \n" +
+  "        ca-certificates curl netcat-openbsd p11-kit unzip zip ${docker_base == 'cloud' ? 'wget' : '' } && \n" +
   "      ${package_manager} clean && \n" +
   "      rm -rf /var/lib/apt/lists/*"
 ) %>
 
-RUN groupadd -g 1000 elasticsearch && \\
-    adduser --uid 1000 --gid 1000 --home /usr/share/elasticsearch elasticsearch && \\
-    adduser elasticsearch root && \\
+RUN userdel -r ubuntu && \\
```

Comment:
ubuntu 24.04 provides a default 'ubuntu' user. best practices I found suggest to just remove it before adding our own user.